.vscode/extensions.json

```diff
@@ -5,11 +5,11 @@
     "editorconfig.editorconfig",
     "esbenp.prettier-vscode",
     "dbaeumer.vscode-eslint",
-    "dorzey.vscode-sqlfluff",
     "ms-ossdata.vscode-pgsql",
     "ms-python.python",
     "ms-python.vscode-pylance",
     "redhat.vscode-yaml",
+    "sqlfluff.vscode-sqlfluff",
```

Comment:
This is the correct extension, the other one isn't receiving regular updates.

Reply:
That's not the VSCode extension registry.

Reply:
Reverted in 942ac91d6bc23ac4b24ceb786cf4f4e7bd04a047.